vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/ConfigurationBusiness.java

```diff
@@ -386,7 +386,7 @@ private User signin(String email, String password, boolean resetSession)
         }
     }
 
-    public User getNewUser(String email, String firstName, String lastName) {
+    public User getNewUser(String email, String firstName, String lastName, String egi_affiliation) {
```

Comment:
We are in internal VIP business class that must not have link with egi
Just name the argument/parameter `institution` everywhere in this class